library/alloc/src/vec/peek_mut.rs

```diff
@@ -11,18 +12,18 @@ use crate::fmt;
 ///
 /// [`peek_mut`]: Vec::peek_mut
 #[unstable(feature = "vec_peek_mut", issue = "122742")]
-pub struct PeekMut<'a, T> {
-    vec: &'a mut Vec<T>,
+pub struct PeekMut<'a, T, A: Allocator> {
+    vec: &'a mut Vec<T, A>,
 }
 
 #[unstable(feature = "vec_peek_mut", issue = "122742")]
-impl<T: fmt::Debug> fmt::Debug for PeekMut<'_, T> {
+impl<T: fmt::Debug, A: Allocator> fmt::Debug for PeekMut<'_, T, A> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         f.debug_tuple("PeekMut").field(self.deref()).finish()
     }
 }
 
-impl<'a, T> PeekMut<'a, T> {
+impl<'a, T, A: Allocator> PeekMut<'a, T, A> {
     pub(crate) fn new(vec: &'a mut Vec<T>) -> Option<Self> {
```

Comment:
You'll see CI failing in a few minutes because of this 😄:
```suggestion
    pub(crate) fn new(vec: &'a mut Vec<T, A>) -> Option<Self> {
```

Reply:
Sigh, I had that, but I failed at correctly operating git.

Reply:
Ah... I know the struggle... 😄